.github/docker/ubuntu-20.04.Dockerfile

```diff
@@ -1,10 +1,10 @@
-# Copyright (C) 2024 Intel Corporation
+# Copyright (C) 2024-2025 Intel Corporation
```

Comment:
pls order commits and add a proper commit title/message; while we're adding one new docker, the main change is something else 😉 

pls also update the title of the PR, and if ready, please un-draft